src/events/message/index.ts

```diff
@@ -24,45 +27,82 @@ export default class MessageEvent extends Event {
 		}
 
 		if (message.attachments.size > 0) {
-			const file = message.attachments.first(); // only take first attachment as, normally, users cannot send more than one attachment
+			const file =
+				message.attachments.find((attachment) => {
+					if (!attachment.name || attachment.width) {
```

Comment:
```suggestion
					if (attachment.width) {
```
`name` n'est jamais nul

Reply:
`null` ou `""`, c'est falsy (plus le nullish coalescing inutile)

Reply:
un `name` pour valoir `""`.